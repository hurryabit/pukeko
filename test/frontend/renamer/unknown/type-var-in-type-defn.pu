data Typ a =
  | Con b
