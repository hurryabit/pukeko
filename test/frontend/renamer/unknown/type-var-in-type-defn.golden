test/frontend/renamer/unknown/type-var-in-type-defn.pu:1:14: unknown type variable: b
